go.mod

```diff
@@ -20,15 +20,13 @@ require (
 	go.uber.org/zap v1.20.0
 	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
 	golang.org/x/sys v0.0.0-20220209214540-3681064d5158 // indirect
-	google.golang.org/genproto v0.0.0-20220211171837-173942840c17 // indirect
+	google.golang.org/genproto v0.0.0-20220215190005-e57b466719ef // indirect
```

Comment:
This looks like a up-to-date version, the latest commit SHA should be `2d9577e0f42b14380cd86ddd1d9ea1a65e85ef18`